cmd/incusd/network_allocations.go

```diff
@@ -219,9 +219,30 @@ func networkAllocationsGet(d *Daemon, r *http.Request) response.Response {
 			var loadBalancers map[int64]*api.NetworkLoadBalancer
 
 			err = d.db.Cluster.Transaction(r.Context(), func(ctx context.Context, tx *db.ClusterTx) error {
-				loadBalancers, err = tx.GetNetworkLoadBalancers(ctx, n.ID(), false)
+				memberSpecific := false
+				network_id := n.ID()
+				dbLoadBalancers, err := dbCluster.GetNetworkLoadBalancers(ctx, tx.Tx())
+				if err != nil {
+					return err
+				}
 
-				return err
+				loadBalancers = make(map[int64]*api.NetworkLoadBalancer)
+
+				for _, lb := range dbLoadBalancers {
+					// memberSpecific filtering
```

Comment:
Shouldn't it be `if memberSpecific ...`? Also, isn't `memberSpecific` false in this case so no additional filtering needs to be done?